### DISCUSSION TITLE
Request: Build for non-`amd64` architectures

### DISCUSSION THREAD

**Discussion by starthal** [makedeb/prebuilt-mpr#176] [Issues]
Thanks for the Prebuilt-MPR!

I'm not sure I understand the constraints here, but is it possible to include builds for another architecture?

In particular we would like to have `just:arm64` available.

**Comment by hwittenborn** [comment]
Thanks a ton! There isn't anything in particular that should stop the Prebuilt-MPR from supporting other architectures. It's not gonna be huge, but it'll be a bit of a redesign of the Prebuilt-MPR's internals to support other architectures, so I can't guarantee when that'll happen.

I'm currently quite busy juggling around 4 other projects, so I'm not positive when I'll be able to set aside the time to flesh out the implementation for supporting multiple architectures. If you'd be willing to submit a PR I'd be more than glad to help guide you through it though. If not I'll still give it a look when I'm able to though, again I'm just not positive when that might be.

**Comment by hwittenborn** [comment]
Hey @starthal, I've had someone sponsor me to help get this all going, and `arm64` support has been added into the Prebuilt-MPR!

All the packages still have to be built for `arm64` which will probably take a few days, but the sponsor was wanting a few packages in particular which will result in `just` hopefully being done by the end of today. I'll send a message once I can confirm that's all done, but I thought I'd send a heads up about progress being made :).
